docker-compose.yml

```diff
@@ -47,6 +47,22 @@ bionic:
     - mongodb
     - postgres
 
+el8:
+  image: packagingrunner:trusty
+  extends:
+    file: docker-compose.override.yml
+    service: suite-compose
+  environment:
+    - BUILDNODE=centos8build
+    - TESTNODE=centos8test
```

Comment:
Compare with `bionic` env, need to add `- ST2_PACKAGES=st2` so we avoid building mistral in any case.